api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -250,6 +252,14 @@ public DbBillingProjectBufferEntry assignBillingProject(DbUser dbUser) {
 
     fireCloudService.addOwnerToBillingProject(
         dbUser.getUsername(), bufferEntry.getFireCloudProjectName());
+
+    if (workbenchConfigProvider.get().featureFlags.enableLazyPerimeterAssignment
+        && bufferEntry.needsPerimeterAssignment()) {
+      fireCloudService.addProjectToServicePerimeter(
+          workbenchConfigProvider.get().firecloud.vpcServicePerimeterName,
```

Comment:
So right now we're just allowing one at a time?

Reply:
yes - still strictly Registered Tier for now